config/atmos_configs/aquaplanet_diagedmf.yml

```diff
@@ -0,0 +1,29 @@
+job_id: "aquaplanet_diagedmf"
```

Comment:
I will alphabetize the aquaplanet configs so we can more easily compare to the amip ones